modules/transcription-service-ibm-watson-impl/src/main/java/org/opencastproject/transcription/ibmwatson/IBMWatsonTranscriptionService.java

```diff
@@ -206,14 +224,29 @@ public void activate(ComponentContext cc) {
       enabled = OsgiUtil.getOptCfgAsBoolean(cc.getProperties(), ENABLED_CONFIG).get();
 
       if (enabled) {
-        // User name (mandatory)
-        user = OsgiUtil.getComponentContextProperty(cc, IBM_WATSON_USER_CONFIG);
-        // Password (mandatory)
-        psw = OsgiUtil.getComponentContextProperty(cc, IBM_WATSON_PSW_CONFIG);
-        logger.info("Using transcription service at {} with username {}", IBM_WATSON_SERVICE_URL, user);
-
-        // Language model to be used
-        Option<String> modelOpt = OsgiUtil.getOptCfg(cc.getProperties(), IBM_WATSON_MODEL);
+        // Service url (optional)
+        Option<String> urlOpt = OsgiUtil.getOptCfg(cc.getProperties(), IBM_WATSON_SERVICE_URL_CONFIG);
+        if (urlOpt.isSome()) {
+          watsonServiceUrl = urlOpt.get() + API_VERSION;
```

Comment:
UrlSupport.concat would remove a lot of the `/`s in the constants above.

Reply:
Code updated. Please confirm that the change made was what you meant.